### ISSUE TITLE
Bootstrap fails in the middle of process if pg_buffercache extension does not exist on a server and cannot be installed

### ISSUE THREAD

**Issue by kuzmindb** [postgrespro/mamonsu#153]
Bootstrap fails in the middle of process if pg_buffercache extension does not exist on a server and cannot be installed. Even if pg_buffercache is disabled in agent.conf.
BTW, as a result, the bootstrap leaves target db partially bootstrapped, which leads to various errors in mamonsu log (because the agent has detected bootstrap):
```
[INFO] 2021-02-16 22:52:38,603 - PGSQL-(host=localhost db=mamonsu user=postgres port=5432)      -       Found mamonsu bootstrap
[INFO] 2021-02-16 22:52:38,749 - XLOG   -       started ...
[ERROR] 2021-02-16 22:52:38,754 - XLOG  -       catch error: {'S': 'ERROR', 'V': 'ERROR', 'C': '42883', 'M': 'function public.mamonsu_count_wal_lag_lsn() does not exist', 'H': 'No function matches the given name and argument types. You might need to add explicit type casts.', 'P': '78', 'F': 'parse_func.c', 'L': '621', 'R': 'ParseFuncOrCo
```

**Comment by casperr0** [comment]
Thank you for the issue
We will fix this in the next version

**Comment by casperr0** [comment]
This is fixed in 2.7.0 relese
